dev/tests/integration/testsuite/Magento/UrlRewrite/_files/url_rewrite.php

```diff
@@ -64,6 +64,26 @@
     ->setStores([$storeID, $secondStoreId]);
 $pageResource->save($page);
 
+$page = $objectManager->create(Page::class);
```

Comment:
Looks like you need also update `url_rewrites_rollback.php` file in the same directory

Reply:
@ihor-sviziev I did changes, but in url_rewrit**e**_rollback.php (url_rewrit**es**_rollback.php belongs to  url_rewrites.php fixture).